tools/test_torchserver.py

```diff
@@ -0,0 +1,57 @@
+import base64
+from argparse import ArgumentParser
+from io import BytesIO
+
+import matplotlib.pyplot as plt
+import mmcv
+import requests
+
+from mmseg.apis import inference_segmentor, init_segmentor
+
+
+def parse_args():
+    parser = ArgumentParser()
```

Comment:
Can add some description。
Refer to https://github.com/open-mmlab/mmsegmentation/blob/master/tools/model_converters/mit2mmseg.py#L62.

Reply:
OK